app/views/challenges/external.html.haml

```diff
@@ -0,0 +1,56 @@
+#challenge-external-page
+  %section.top-section
+    .lg-container
+      .flex.flex-wrap.justify-center.mb-32.gap-2.lg:gap-4.mx-auto{ class: 'max-w-[1000px]' }
+        - Track.active.map(&:icon_url).shuffle.each do |icon_url|
+          = image_tag icon_url, alt: '', role: :presentation, class: "c-icon c-track-icon h-[24px] w-[24px] lg:h-[48px] lg:w-[48px]"
+
+      %h1.text-h0.mb-16 The #12in23 Challenge
+      %p.mb-12
+        Expand your horizons by trying out
+        %strong.font-semibold 12 different programming languages
+        in 2023.
+      %p
+        Go old-school with COBOL, cutting edge with Unison and esoteric with Prolog. Explore low-level code with Assembly, expressions with a Lisp and functional with Haskell!
```

Comment:
Maybe change "and" to "or" to make it clear to students that these are choices, not requirements.

```suggestion
        Go old-school with COBOL, cutting edge with Unison or esoteric with Prolog. Explore low-level code with Assembly, expressions with a Lisp or functional with Haskell!
```